dtale/cli/loaders/json_loader.py

```diff
@@ -16,6 +16,7 @@
         name="convert_dates",
         help="comma-separated string of column names which should be parsed as dates",
     ),
+    dict(name="lines", help="Use if the file is a JSONL line file"),
```

Comment:
Since this is a boolean field we can make it be a click flag.  This will allow us to call the json loader from the CLI and if `--json-lines` is preset it will set `lines=True` otherwise it will be `lines=False`. 
```suggestion
    dict(name="lines", help="Use if the file is a JSONL line file", is_flag=True),
```
In the essence of time I'll make an updated commit myself (and i'll update the author to you so you get the credit).  Thanks for the PR! 🙏

Reply:
Thank you! I didn't see the is_flag option. Looks great!